File: lua/test/exists_test.lua
-- BluefinShieldconex SDK exists test

local sdk = require("bluefin-shieldconex_sdk")

describe("BluefinShieldconexSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: csharp/test/ExistsTest.cs
// BluefinShieldconex SDK exists test.

using Xunit;

using BluefinShieldconexSdk;

namespace BluefinShieldconexSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinShieldconexSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}
